src/golang/cmd/server/middleware/usage/utils.go

```diff
@@ -0,0 +1,24 @@
+package usage
```

Comment:
nit: Is there a more specific file name we can use than `utils.go`. I'm trying to get us off of dumping everything in utils.go/py if possible - have been using the term `models` for structs like these but not sure.

Reply:
`models.go` sounds good